instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -432,3 +438,28 @@ def test_tracer_provider_traced(self):
         self.assertEqual(
             span.resource.attributes["resource-key"], "resource-value"
         )
+
+    
+    def test_django_with_wsgi_instrumented(self):
+        """ 
+        This test is related to https://github.com/open-telemetry/opentelemetry-python-contrib/issues/448
+        Here we have enabled django as well as wsgi instrumentation.
+        and on sending the request, it should give us the two spans with parent-child relationship.
+        """
+        application = get_wsgi_application()
+        application = OpenTelemetryMiddleware(application, tracer_provider=self.tracer_provider)
+        environ = RequestFactory()._base_environ(
+            PATH_INFO="/span_name/1234/",
+            CONTENT_TYPE="text/html; charset=utf-8",
+            REQUEST_METHOD="GET"
+        )
+        response_data = {}
+        def start_response(status, headers):
+            response_data["status"] = status
+            response_data["headers"] = headers
+
+        resp = next(application(environ, start_response))
```

Comment:
Please use the test client here.

Reply:
I have tried but it is not able to create multiple spans when test client is being used